Automattic/wp-calypso
# Update copy in plan renewal message

### Changes proposed in this Pull Request

This PR updates the warning notice shown when user has a legacy Jetpack plan about to expire.

Fixes 1169247016322522-as-1189044397284249

### Implementation notes

I stripped down `PlanRenewalMessage` since the `else` condition and the external link weren't relevant anymore.

### Testing instructions

- Choose a self-hosted Jetpack site with a legacy plan that's about to expire (you can update the expiration date in the store admin)
- Visit `/plans/my-plan/:site` and check that the notice matches the capture below (and that the link works)
- Visit `/me/purchases/:site/:purchase-id` and check that the notice matches the capture below (and that the link works)
- Visit `/plans/:site` and check  that the description in the card matches the capture below

### Screenshots

#### My plan
<img width="613" alt="Screen Shot 2020-09-09 at 12 37 48 PM" src="https://user-images.githubusercontent.com/1620183/92628849-6e251c80-f29b-11ea-806a-21dc2fda4bc5.png">

#### Purchase
<img width="616" alt="Screen Shot 2020-09-09 at 12 37 42 PM" src="https://user-images.githubusercontent.com/1620183/92628864-71b8a380-f29b-11ea-944a-c9e37fe7765b.png">


#### Product card
<img width="337" alt="Screen Shot 2020-09-09 at 12 43 39 PM" src="https://user-images.githubusercontent.com/1620183/92628884-78471b00-f29b-11ea-9aa9-346f24cfd996.png">
